app/src/main/java/org/onionshare/android/ShareManager.kt

```diff
@@ -61,8 +64,7 @@ class ShareManager @Inject constructor(
 
         // not supporting selecting entire folders with sub-folders
         val filesAdded = fileManager.addFiles(uris, shareState.value.files)
-        val totalSize = filesAdded.files.sumOf { it.size }
-        _shareState.value = ShareUiState.FilesAdded(filesAdded.files, totalSize)
+        _shareState.value = ShareUiState.FilesAdded(filesAdded.files, filesAdded.files.totalSize)
```

Comment:
Nitpick: looks like we could save some code by removing the totalSize field from all these states, as we can calculate it when needed using the List<SendFile> extension method.

Reply:
Yeah that's a good point. I did some work towards that here, but should follow through with it indeed. Pushed a new commit for it.